cli/dts/lib.deno.unstable.d.ts

```diff
@@ -617,12 +619,53 @@ declare namespace Deno {
     SIGINFO = 29,
     SIGUSR1 = 30,
     SIGUSR2 = 31,
+
+    SIGBREAK = 0,
   }
 
+  enum WindowsSignal {
```

Comment:
It's a bit over the top to have platform specific types in the API.
We should change this to `enum Signal {` and remove the `type Signal = ...` below, and remove the `LinuxSignal` and `MacOSSignal` types.

@bartlomieju, @kt3k would you agree?

Reply:
> 在API中拥有特定于平台的类型有些困难。
> 我们应该将其更改为`enum Signal {`并删除`type Signal = ...`以下内容，并删除`LinuxSignal`和`MacOSSignal`类型。
> 
> @ bartlomieju，@ kt3k你同意吗？

I think this is a bit difficult, because there are some signals that are different between Linux and macOS. For example, SIGBUS is 7 in Linux, but 10 in macOS.